auth/auth.go

```diff
@@ -222,7 +265,8 @@ func (r *RenewingChecker) renewVpnCredentials(data *config.TokenData) error {
 	return nil
 }
 
-func (r *RenewingChecker) updateVpnExpirationDate(data *config.TokenData) error {
+// fetchSaveServices fetches services and updates data appropriately
+func (r *RenewingChecker) fetchSaveServices(userId int64, data *config.TokenData) error {
```

Comment:
I dont see use results of this function - it fetches services and saves, but saved services are not used, this data is fetched again regardless of what is saved?

Reply:
It's still used for fetching regular(not dedicated) VPN service.